kotlin-sqlobject/src/main/kotlin/org/jdbi/v3/sqlobject/kotlin/RegisterKotlinMapper.kt

```diff
@@ -0,0 +1,30 @@
+package org.jdbi.v3.sqlobject.kotlin
+
+import org.jdbi.v3.sqlobject.config.ConfiguringAnnotation
+import org.jdbi.v3.sqlobject.config.RegisterKotlinMappers
+import java.lang.annotation.ElementType
+import java.lang.annotation.Retention
+import java.lang.annotation.RetentionPolicy
+import java.lang.annotation.Target
+import kotlin.reflect.KClass
+
+/**
+ * Registers a KotlinMapper for a specific kotlin class
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@ConfiguringAnnotation(RegisterKotlinMapperImpl::class)
+@Target(ElementType.TYPE, ElementType.METHOD)
+@Repeatable
+@java.lang.annotation.Repeatable(RegisterKotlinMappers::class)
```

Comment:
Both, kotlin's `@Repeatable` and java's `@Repeatable` are needed for the time being because kotlin doesn't support repeatable annotations yet. Presumably, once it does, we can get rid of the plural annotation and its corresponding `RegisterKotlinMappersImpl`